### PR TITLE
Feature/new glasses intro

### PR DESCRIPTION
**Pull request by Blizzardsev**
Adds the following:
- New intro event: `event_eyewear_problems`
- New glasses for outfits system; circular, rectangular and sunglasses
- Updated remaining MP3 sound effects to OGG format
- Slight tweak to a farewell

### CODE REVIEW

File: game/outfits.rpy [hunk 1]
```diff
@@ -1120,14 +1120,38 @@ init -1 python in jn_outfits:
...
+    ))
+    __register_wearable(JNEyewear(
+        reference_name="jn_eyewear_rectangular_glasses_black",
+        display_name="Black rectangular glasses",
+        unlocked=False,
         is_jn_wearable=True
     ))
     __register_wearable(JNEyewear(
-        reference_name="jn_eyewear_heart_frames",
-        display_name="Pink heart glasses",
```
Comment (RaionArt) [inline game/outfits.rpy:1130]: Do you plan to delete them totally or should I tune the pink a bit down and we can use them later?

**Review by RaionArt — APPROVED** [review]

File: game/outfits.rpy (hunk 1 above)
Comment (Blizzardsev) [reply game/outfits.rpy:1130]: Good point - I'll leave that to your best judgment

File: game/script-events.rpy [hunk 2]
```diff
@@ -29,7 +54,7 @@ init python in jn_events:
             is_seen=False,
             **kwargs
         )
-
+        return None
```
Comment (TRIDENT1313) [inline game/script-events.rpy:57]: Since None is test code we want to remove it.
```suggestion
```

**Review by TRIDENT1313 — CHANGES_REQUESTED** [review]

**Review by TRIDENT1313 — APPROVED** [review]

File: game/outfits.rpy (hunk 1 above)
Comment (Blizzardsev) [reply game/outfits.rpy:1130]: Did we have an idea of where to go with this?

Comment (RaionArt) [reply game/outfits.rpy:1130]: If you want you can delete them from the quote. We still have the files on drive in case we want to bring them back we can.

**Pull request merged**